src/main/java/org/radarcns/management/web/rest/util/HttpUtil.java

```diff
@@ -0,0 +1,23 @@
+package org.radarcns.management.web.rest.util;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class HttpUtil {
+
+    public static boolean isReachable(URL urlServer) {
+        try {
+            HttpURLConnection urlConn = (HttpURLConnection) urlServer.openConnection();
+            urlConn.setConnectTimeout(3000); //<- 3Seconds Timeout
+            urlConn.connect();
+            return urlConn.getResponseCode() == 200;
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Perhaps it is sufficient to do a `log.warn("Server {} is unreachable: {}", urlServer.toString(), e.getMessage())`, a stacktrace less nice to deal with in the logs.